crates/api/api_crud/src/post/update.rs

```diff
@@ -141,7 +141,8 @@ pub async fn update_post(
     scheduled_publish_time_at,
     ..Default::default()
   };
-  post_form = plugin_hook_before("before_update_local_post", post_form).await?;
+  // TODO this is currently broken
+  // post_form = plugin_hook_before("before_update_local_post", post_form).await?;
```

Comment:
This is still causing API test failures due to some bug in our plugin code. Commenting it out for now so we can get CI passing.